docs/how-to/uc-image-creation.rst

```diff
@@ -47,6 +41,13 @@ Below are example model assertions, describing the Ubuntu Core image content for
         The ``console-conf`` snap added here is only to allow interactive user and network configuration.
```

Comment:
Could you also include the object defining console-conf snap to `emphasize-lines`? It should be highlighted as that is a change compared to the original model assertion.

Reply:
Done!